benchmarks/index.md

```diff
@@ -24,7 +24,7 @@ At the macroscale, on the other hand, materials properties are
 averaged over many domains (e.g., compositional, structural, magnetic,
 or ferroelectric domains). Central entities at the macroscale are
 usually materials properties that describe a response to some
-macroscopic external forces, for example conductivity, elastic
+macroscopic external forces, e.g., conductivity, elastic
```

Comment:
```*e.g.*```, since *exempli gratia* is not an English phrase

Reply:
the asterisks will add formatting?  I actually don't know how to preview the changes as a web page...

Reply:
Yes, the ```md``` extension is for markdown; use [GitHub flavored syntax](https://guides.github.com/features/mastering-markdown/). To preview the website, or check your formatting, I would create a [gist](https://gist.github.com) with your content and render it. For example, https://gist.github.com/tkphd/5b57a930d5b4ed46e0a9cac805cfa6d6

Reply:
oooh thanks.

Reply:
Also, if you happen to use Atom as your editor, there's a plugin to preview markdown content that I've found to be really helpful:
https://atom.io/packages/markdown-preview-plus

Reply:
Oh brilliant, thanks @stvdwtt!